lib/renderer/extensions/web-navigation.ts

```diff
@@ -1,23 +1,20 @@
-'use strict'
-
-const Event = require('@electron/internal/renderer/extensions/event')
+import { Event } from '@electron/internal/renderer/extensions/event'
+import { IpcMainEvent } from 'electron'
 const { ipcRendererInternal } = require('@electron/internal/renderer/ipc-renderer-internal')
 
 class WebNavigation {
-  constructor () {
-    this.onBeforeNavigate = new Event()
-    this.onCompleted = new Event()
+  private onBeforeNavigate = new Event()
+  private onCompleted = new Event()
 
-    ipcRendererInternal.on('CHROME_WEBNAVIGATION_ONBEFORENAVIGATE', (event, details) => {
+  constructor () {
+    ipcRendererInternal.on('CHROME_WEBNAVIGATION_ONBEFORENAVIGATE', (event: IpcMainEvent, details: any) => {
       this.onBeforeNavigate.emit(details)
     })
 
-    ipcRendererInternal.on('CHROME_WEBNAVIGATION_ONCOMPLETED', (event, details) => {
+    ipcRendererInternal.on('CHROME_WEBNAVIGATION_ONCOMPLETED', (event: IpcMainEvent, details: any) => {
       this.onCompleted.emit(details)
     })
   }
 }
 
-exports.setup = () => {
-  return new WebNavigation()
-}
+exports.setup = () => new WebNavigation()
```

Comment:
```suggestion
export const setup = () => new WebNavigation()
```

I would also question this method, seems a bit useless, why not just export the `class` 😆 (I know it was like this before but hey)